### ISSUE TITLE
Unable to override ui_component

### ISSUE THREAD

**Issue by VZeroCool** [KishanSavaliya/magehelper-magento2-overrite-backend-theme#1]
I've used your theme to override a xml from Magento_InventoryAdminUi but not working any ideas ? 

File I'm trying is inventory_source_listing.xml

**Comment by KishanSavaliya** [comment]
Where you're trying to add that XML file in theme ?

You can add that file here

> app/code/MageHelper/BackendTheme/view/adminhtml/ui_component/inventory_source_listing.xml

Or you can add that here in design directory as well

> app/design/adminhtml/MageHelper/BackendTheme/Magento_InventoryAdminUi/ui_component/inventory_source_listing.xml

Hope this will help you!

**Comment by VZeroCool** [comment]
Yes placing in your custom theme worked partially, Actually I'm trying to remove **Add New Source** button, After copying file to backend theme I can able to override text but a I cant able to remove button node from XML. Why? Any ideas?

```
   <settings>
        <!-- <buttons>
            <button name="add">
                <url path="*/*/new"/>
                <class>primary</class>
                <label translate="true">Add New Source444</label>
            </button>
        </buttons> -->
        <spinner>inventory_source_listing_columns</spinner>
        <deps>
            <dep>inventory_source_listing.inventory_source_listing_data_source</dep>
        </deps>
  </settings>
```
